lametro/people.py

```diff
@@ -102,7 +102,7 @@ def scrape(self):
             members[member] = p
 
         for body in self.bodies():
-            if body['BodyTypeId'] == body_types['Committee']:
+            if body['BodyTypeId'] in (body_types['Committee'], body_types['Independent Taxpayer Oversight Committee']):
```

Comment:
this looks good. not a big deal when you have only two items, but a set instead of a tuple could be good here.

Reply:
Interesting. Can you say more on why that is?

Reply:
sets are the data structures tailored for checking for membership inclusion. it really doesn't matter for two items, but

1. checking membership in sets will be O(1) while checking membership in tuple will be O(N) (where N is the number of elements)
2. since sets are the data structures that are excellent for checking membership, I personally have a mild expectation that they will be used when that's the only thing we want from a sequence, and I think a bit more when that expectation is violated. For example, why is this a tuple, is it because the elements are not hashable? This is probably idiosyncratic to me.

anyway, when there are only two items it's certainly a matter of stylistic preference.

Reply:
Ah, got it. Thanks!